src/Abstractions/NexusMods.Abstractions.Games.Diagnostics/Diagnostic.cs

```diff
@@ -17,6 +17,14 @@ public record Diagnostic
     /// </remarks>
     public required DiagnosticId Id { get; init; }
 
+    /// <summary>
+    /// Gets the title of the diagnostic.
+    /// </summary>
+    /// <remarks>
+    /// This must not contain any fields.
+    /// </remarks>
+    public required string Title { get; init; }
```

Comment:
The comment doesn't explain the difference with summary enough.
Title should be used to identify the type of problem, without details of the specific diagnosed instance.